packages/hint-scoped-svg-styles/README.md

```diff
@@ -0,0 +1,131 @@
+detect if any style (CSS rule or selector) inside `<svg>` selects
+elements outside it.
+
+## What does the hint check?
+
+This hint checks if styles inside SVG affect any other elements outside
+it. If any such style is found, it reports the CSS rule that is
+affecting and html elements that are being affected.
+
+### Examples that **trigger** the hint
+
+Any style rule inside SVG that selects elements outside it will trigger
+the hint.
+
+Example:
+
+```html
+<html>
+    <body>
+        <h1 class="my-style">Heading</h1>
+
+        <svg>
+            <style>
+                .my-style {
+                    opacity: 0.5;
+                }
+            </style>
+            <text class="my-style">SVG text</text>
+        </svg>
+    </body>
+</html>
+```
+
+### Examples that **pass** the hint
+
+If in document, all styles inside SVGs are scoped to SVG only and are
+not affecting any element outside it, hint will pass.
+
+Example:
+
+```html
+<html>
+    <body>
+        <h1 class="html-style">Heading</h1>
+
+        <svg>
+            <style>
+                .svg-style {
+                    opacity: 0.5;
+                }
+            </style>
+            <text class="svg-style">SVG text</text>
+        </svg>
+    </body>
+</html>
+```
+
+## Can the hint be configured?
+
+This hint can be configured to limit the number of HTML elements reported
+per CSS rule. If the hint finds an affecting CSS rule, it generates one report
+related to that rule and an additional report for each HTML element
+matched by that rule.
+
+If the `maxReportsPerCSSRule` option is passed to this hint, it will limit the
+number of reports related to affected elements, but reports related to
+CSS rule will still be there.
+
+### How to pass `maxReportsPerCSSRule` option to hint
+
+`maxReportsPerCSSRule` can be added in `.hintrc` as given below:
+
+```json
+    ...
+    "hints": {
+            "scoped-svg-styles": [
+                    "warning", {
+                        "maxReportsPerCSSRule": 5
+                    }
+            ]
+        },
+    ...
+```
+
+In above example, For every affecting rule, there can be a maximum of six
+reports. One report related to the CSS rule itself and a maximum of
+five reports related to affected elements.
+
+## How to use this hint?
+
+To use it you will have to install it via `npm`:
+
+```bash
+npm install @hint/hint-scoped-svg-styles
+```
+
+Note: You can make `npm` install it as a `devDependency` using the `--save-dev`
+parameter, or to install it globally, you can use the `-g` parameter. For
+other options see
+[`npm`'s documentation](https://docs.npmjs.com/cli/install).
+
+And then activate it via the [`.hintrc`][hintrc]
+configuration file:
+
+```json
+{
+    "connector": {...},
+    "formatters": [...],
+    "parsers": [...],
+    "hints": {
+        "scoped-svg-styles": "error"
+    },
+    ...
+}
+```
+
+## Further Reading
+
+What can the user read to know more about this subject?
```

Comment:
You can just remove this text (and the Further Reading header) if you don't have any additional links to add.

```suggestion
```

Reply:
Ok for now I will be removing it. But I have it in todo to find some useful resource related to this hint. Like tools to fix this issue. There are some tools to convert `<style>` inside SVG to inline style attributes.
After this week work, I will be updating it.